cogs/admin.py

```diff
@@ -192,50 +192,60 @@ async def alignmentList(self, ctx):
         player_list = list(db.players.find(
                {"Alignment": {"$exists": True}})
             )
-        await ctx.channel.send(content="\n".join([f"{x}: {y}" for x,y in dict(collections.Counter(sorted(list([x["Alignment"].replace("\"", "") for x in player_list])))).items()]))  
-    
+            
+        contents = []
+        alignment=""
+        for s in player_list:
+            alignment += f"{s['Alignment']}\n"
+        contents.append(("Alignment", alignment, False))
+        await paginate(ctx, self.bot, f"Alignments List", contents=contents, separator="\n")
+        
+        
     @commands.command()
     @commands.has_any_role("Mod Friend")
     async def reflavorList(self, ctx):
         player_list = list(db.players.find(
                {"Reflavor": {"$exists": True}})
             )
-        rfarray = list([line["Reflavor"] for line in player_list]) #puts all the 
-        raceList = []
-        classList = []
-        backgroundList = []
+        rfarray = list([line["Reflavor"] for line in player_list]) #puts all the reflavors in a list
+        raceContents=[]
+        classContents=[]
+        backgroundContents=[]
+        races=""
+        classes=""
+        backgrounds=""
         i = 0
         while i < len(rfarray):
             if 'Race' in rfarray[i]:
-                raceList.append(rfarray[i]['Race'])
+                races += f"{rfarray[i]['Race']}\n"
             if 'Class' in rfarray[i]:
-                classList.append(rfarray[i]['Class'])
+                classes += f"{rfarray[i]['Class']}\n"
             if 'Background' in rfarray[i]:
-                backgroundList.append(rfarray[i]['Background'])
+                backgrounds += f"{rfarray[i]['Background']}\n"
             i += 1
-        await ctx.channel.send("**Races:**")
-        await ctx.channel.send(content="\n".join([f"{x}: {y}" for x,y in dict(collections.Counter(sorted(list([x.replace("\"", "") for x in raceList])))).items()]))
-        await ctx.channel.send("**Classes:**")
-        await ctx.channel.send(content="\n".join([f"{x}: {y}" for x,y in dict(collections.Counter(sorted(list([x.replace("\"", "") for x in classList])))).items()]))
-        await ctx.channel.send("**Backgrounds:**")
-        await ctx.channel.send(content="\n".join([f"{x}: {y}" for x,y in dict(collections.Counter(sorted(list([x.replace("\"", "") for x in backgroundList])))).items()]))  
+        raceContents.append(("Race", races, False))
+        classContents.append(("Class", classes, False))
+        backgroundContents.append(("Background", backgrounds, False))
+            
+        await paginate(ctx, self.bot, f"Race Reflavor List", contents=raceContents, separator="\n")
+        await paginate(ctx, self.bot, f"Class Reflavor List", contents=classContents, separator="\n")
+        await paginate(ctx, self.bot, f"Background Reflavor List", contents=backgroundContents, separator="\n")
```

Comment:
These should also be one paginate or else it is going to have 3 menus after each other and you can only get to next once the previous one times out